sample/Ookii.Dialogs.Wpf.Sample/Properties/AssemblyInfo.cs

```diff
@@ -1,62 +0,0 @@
-﻿using System.Reflection;
-using System.Runtime.InteropServices;
-using System.Windows;
-
-// General Information about an assembly is controlled through the following 
-// set of attributes. Change these attribute values to modify the information
-// associated with an assembly.
-[assembly: AssemblyTitle("Ookii.Dialogs.Wpf.Sample")]
-[assembly: AssemblyDescription("")]
-
-#if DEBUG
-[assembly: AssemblyConfiguration("Debug")]
-#else
-[assembly: AssemblyConfiguration("Release")]
-#endif
-
-[assembly: AssemblyCompany("Ookii.org")]
-[assembly: AssemblyProduct("Ookii.Dialogs.Wpf")]
-[assembly: AssemblyCopyright("Copyright (c) Sven Groot (Ookii.org) 2009")]
-[assembly: AssemblyTrademark("")]
-[assembly: AssemblyCulture("")]
-[assembly: System.CLSCompliant(true)]
```

Comment:
Is `CLSCompliant` being automatically generated now through the new SDK project?